crates/iota-graphql-client/examples/prepare_send_iota_multi.rs

```diff
@@ -71,7 +71,7 @@ async fn main() -> Result<()> {
         client
             .reference_gas_price(None)
             .await?
-            .context("missing ref gas price")?,
+            .ok_or_eyre("missing ref gas price")?,
```

Comment:
I like `context` much more :(

Reply:
Could actually be used with an additional trait import
```
NOTE: However, to help with porting we do provide a ContextCompat trait which implements context for options which you can import to make existing .context calls compile.
```
https://crates.io/crates/eyre